azurerm/resource_arm_function_app.go

```diff
@@ -66,6 +66,12 @@ func resourceArmFunctionApp() *schema.Resource {
 				Optional: true,
 			},
 
+			"built_in_logging_enabled": {
```

Comment:
we've got previous examples of using `builtin` vs `built_in` - such that I think this'd make more sense as:

```suggestion
			"enable_builtin_logging": {
```

Reply:
can we also update the documentation to include this new field?